src/BugsnagUnity/Payload/StackTraceLine.cs

```diff
@@ -31,7 +31,7 @@ internal StackTrace(StackFrame[] stackFrames)
 
         internal StackTrace(string stackTrace, StackTraceFormat format)
         {
-            string[] lines = stackTrace.Split(new[] { System.Environment.NewLine },
+            string[] lines = stackTrace.Split(new[] {"\r\n","\r","\n", System.Environment.NewLine },
```

Comment:
Is this guaranteed to split in precedence order of the array elements?  Or could a `\r\n` sequence end up being split twice, resulting in a blank entry?